catala/panic/panic.md

```diff
@@ -1,16 +1,20 @@
 De sobte no pots respirar, els pulmons se't fan petits i no t'entra l'aire.
 
-Notes com tot el cos se't paralitza de por. 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.